.github/workflows/check.yml

```diff
@@ -62,6 +62,34 @@ jobs:
     - name: RSpec run
       run: bundle exec rspec
 
+  typecheck:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Update Ubuntu package repository
+      run: sudo apt-get update
+    - name: Setup sqlite-devel
+      run: sudo apt-get -y install libsqlite3-dev
+    - name: Setup libcurl-devel
+      run: sudo apt-get -y install libcurl4 libcurl3-gnutls libcurl4-openssl-dev
+
+    - name: Set up Ruby 3.1.4
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 3.1.4
+        bundler-cache: false
+
+    - name: Install dependencies
+      run: bundle install
+
+    - name: Run sorbet type check
+      run: bundle exec srb typecheck
+
```

Comment:
We already have different jobs for `rspec` and `rubocop`, but they actually should be one job; we can save many lines and computation time for setup in that way.

Reply:
Add `continue-on-error` for each of typecheck, rspec, and rubocop. You can later check if all of the steps succeeded.